File: 1/python/funcs.py
def find_multiples(k, n):
    """
    returns iterator of all multiples of k in n, up to n
    not ordered
    """
    seen = set()
    for item in (l for i in k for l in range(i, n, i)):
        if item not in seen:
            seen.add(item)
            yield(item)

File: 1/JS/funcs.js
exports.find_multiples =  function find_multiples(k, n) {
  multiples = [];
  for (let divisor of k) {
    for (let i = divisor; i < n; i += divisor) {
      multiples.push(i);
    }
  }
  return [...new Set(multiples)].sort();
};